EIPS/eip-649.md

```diff
@@ -0,0 +1,61 @@
+## Preamble
+
+    EIP: 649
+    Title: Metropolis Difficulty Bomb Delay and Issuance Reduction
+    Authors: Afri Schoedon, Vitalik Buterin
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-06-21
+    Replaces: 186
+
+## Simple Summary
+The average block times are increasing due to the difficulty bomb (also known as the "_ice age_") slowly accelerating. This EIP proposes to delay the difficulty bomb for approximately one and a half year and to reduce the block rewards with the Byzantium fork, the first part of the Metropolis fork.
+
+## Abstract
+After `BYZANTIUM_FORK_BLKNUM` the client will calculate the difficulty based on a fake block number suggesting the client that the difficulty bomb is adjusting around 3 million blocks later than previously specified with the Homestead fork. Furthermore, block rewards will be adjusted to a base of 3 ETH, uncle and nephew rewards will be adjusted accordingly.
+
+## Motivation
+The Casper development and switch to proof-of-stake is delayed, the Ethash proof-of-work should be feasible for miners and allow sealing new blocks every 15 seconds on average for another one and a half years. With the delay of the ice age, there is a desire to not suddenly also increase miner rewards. The difficulty bomb has been known about for a long time and now it's going to stop from happening. In order to maintain stability of the system, a block reward reduction that offsets the ice age delay would leave the system in the same general state as before. Reducing the reward also decreases the likelihood of a miner driven chain split as Ethereum approaches proof-of-stake.
+
+## Specification
+#### Relax Difficulty with Fake Block Number
+For the purposes of `calc_difficulty`, simply replace the use of `block.number`, as used in the exponential ice age component, with the formula:
+
+    fake_block_number = block.number - 3_000_000 if block.number >= BYZANTIUM_FORK_BLKNUM else block.number
```

Comment:
During BlockchainTests, Byzantium can start at blocknumber 0 or 2.  `fake_block_number` becomes negative.  Can we make it
```
fake_block_number = min(0, block.number - 3_000_000) if ...
```
?

Reply:
https://github.com/ethereum/EIPs/pull/669#pullrequestreview-57144342